addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -449,7 +449,7 @@ function classToStyle($editable, cssRules) {
         }
         // Apple Mail
         if (node.nodeName === 'TD' && !node.childNodes.length) {
-            writes.push(() => { node.appendChild(document.createTextNode('&nbsp;')); });
+            writes.push(() => { node.appendChild(document.createTextNode('\xA0')); }); // append &nbsp;
```

Comment:
Nice catch ! It looks to be a regression due to removing jQuery in this function at [1] as jQuery actually _did_ the conversion. 

That being said, we prefer to use unicode charaters though so I suggest using `\u00A0` instead. Also, considering the line is already quite long, I suggest putting a more explicit comment above the line. All in all, we get this:
```js 
// Apple Mail
if (node.nodeName === 'TD' && !node.childNodes.length) {
    // Append non-breaking spaces to empty table cells.
    writes.push(() => { node.appendChild(document.createTextNode('\u00A0')); });
}
```

Thanks ! 👍 

[1]: https://github.com/odoo/odoo/commit/18fb205c41c40f8cb726aa18c4f42e34cb39153f#diff-5922d38ff6520249ff5b3a3240eebc350c6f9f4d9f486d512128c23261c45841R369

Reply:
Updated.